.github/workflows/ci.yaml

```diff
@@ -0,0 +1,69 @@
+name: CI
+permissions:
+  contents: read
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+env:
+  CARGO_NET_RETRY: 3
+  RUSTFLAGS: "-D warnings -W unreachable-pub"
+
+jobs:
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust:
+          - stable
+          # XXX: What is the MSRV?

```

Comment:
Not sure what the msrv is, but assume it depends on zbus.

Reply:
"MSRV" is "Minimum Supported Rust Version." For example, if you intend this crate to be compatible with Rust 1.51 and later, then your MSRV would be 1.51 and we'd add "- 1.51.0" here. That way, the CI would verify that we're not using any features from any newer versions of Rust that would break compatibility with 1.51.

If you intend to only support the latest stable version of Rust, then we could just document that with a comment here and remove the " XXX: What is the MSRV?" line.

Reply:
`zbus` doesn't have an MSRV listed and the other crates that come closest are the `RustCrypto` crates. I've picked Rust 1.46 as the MSRV for now since its reasonably old at ~10 months and `secret-service-rs` compiles with it.

It can be bumped in the future with crate version updates.